js/web/lib/wasm/wasm-training-core-impl.ts

```diff
@@ -163,7 +167,9 @@ export const createTrainingSessionHandle = (
     wasm._free(optimizerModelData[0]);
 
     if (sessionOptionsHandle !== 0) {
-      wasm._OrtReleaseSessionOptions(sessionOptionsHandle);
+      if (wasm._OrtReleaseSessionOptions(sessionOptionsHandle) !== 0) {
+        checkLastError('Error occurred when trying to release the session options');
```

Comment:
I know. This is only to consume the return value I have added. Previously _OrtReleaseSessionOptions used to return void. However, because the documentation does not say how to describe SIGNATURE_CONVERSION for void returning function, I made them return a value.